packages/recomponents/src/common/helpers/date.js

```diff
@@ -0,0 +1,13 @@
+import moment from 'moment-timezone';
+import DateTimeFormats from '../datetime-formats';
+
+export default (value, format = DateTimeFormats.date) => {
```

Comment:
I think that `date` is too vague. I would suggest `formattedDatetime` or `formatDatetime`(in case we prefer to always name functions starting with a verb).

Reply:
It is a common helper which is used in other Rebilly frontend projects, I think it will be easier and less confusing in the future to leave it with this naming.

Reply:
In this case doing a "complete refactor-rename" cannot be done in one single step but, we could use a simple [parallel change](https://martinfowler.com/bliki/ParallelChange.html) technique:

Wrapping the current function like this: 
```js
export const formattedDatetime = (value, format = DateTimeFormats.date) => date(value, format)
```
And start replacing step by step from the different clients.